services/web/server/src/simcore_service_webserver/login/routes.py

```diff
@@ -58,4 +59,4 @@ def include_path(tuple_object):
 
     log.debug("Mapped auth routes: %s", "\n".join([pformat(r) for r in routes]))
 
-    return routes
+    return cast(list[web.RouteDef], routes)
```

Comment:
you can avoid the casting by specifying the type to routes when creating the variable.